base/io.jl

```diff
@@ -285,7 +304,7 @@ end
 
 readline() = readline(STDIN)
 readline(s::IO) = readuntil(s, '\n')
-readchomp(x) = chomp!(readall(x))
```

Comment:
Got it, `readchomp` exists because we don't want to export the dangerous `chomp!`. Carry on.